appendix_meta_rules.md

```diff
@@ -102,20 +105,20 @@ The purpose is to cover a detection like:
 
 ### File Inclusion
 
-Sometimes it makes sense to define rules for events in a different file than the correlations, e.g. to make them reusable from multiple correlations or make it possible to use them independently.  
-For this reason, another document type is included for file inclusion.  
-An inclusion can be defined by setting the action attribute to include.  
+Sometimes it makes sense to define rules for events in a different file than the correlations, e.g. to make them reusable from multiple correlations or make it possible to use them independently.
+For this reason, another document type is included for file inclusion.
+An inclusion can be defined by setting the action attribute to include.
 All rules contained in the referenced file are handled as if they were defined in the including file.
 
-###  Filter rules
+### Global Filter rules
 
 The purpose of Filter rules is to apply the same tuning on many rules with the goal to suppress matches of multiple rules. This is most commonly useful for environment specific tuning where a false positive prone application is used in an organization and its false positives are accepted.
-Example: A valid GPO script that triggers X Sigma rules.
+Example: A valid GPO script that triggers multiple Sigma rules.
 
 
 # Correlation rules
 
-All rules in a file, basic event rules as well as correlations, might contain an additional attribute called "generated".  
+All rules in a file, basic event rules as well as correlations, might contain an additional attribute called "generated".
```

Comment:
Should be "generate".

Also: This implies that with this change, no current rules will generate queries (since they don't have that field).

Reply:
This is only meant for rules of type correlation, i.e. having a `correlation: ` section instead of a `dectection:` section.

Reply:
```suggestion
All rules in a file, basic event rules as well as correlations, might contain an additional attribute called "generate".
```

Reply:
About rename: I find it a bit counter-intuitive that this field is called "generate". From a technical perspective it's correct (no queries are generated), but from a high-level user perspective, I don't know what's generated. That it defaults to true (according to documentation below) is also strange / unexpected.
Maybe we could invert it and call it `silent` instead? That would have the "expected" behaviour of defaulting to a zero value (that is, `false`) and it clarifies what this does (if it's set, the rule is silent and does not produce matches).

Reply:
The default behaviour should be that the single rules in a correlation rule are silent. 

Examples:
- The multiple failed logon followed by a successful logon example. You wouldn't want to trigger a rule and see a match for every failed logon. 
- A rule that detects recon activity. You wouldn't want to see matches for every "net" command execution or every "ping". 

So, the default behaviour of the field "silent", if we call it that way, should be "true" and the user can set it to "false" if he wants the single rules in the correlation to show up. 

I wouldn't overcomplicate it with lists of selections. 
But this opinion isn't by far as strong as the one I have on the default behaviour. 
We can allow the user to provide a list of rule `name` that he wants to see matches for. (a list that disables the "silent" for 1-n of the sub rules)

Reply:
The issue that Max raised was that how does that affect existing rules.

My proposal is to define it like this:
> We can switch back to silent is false by default (implicitly). And silent is true if you set it to true explicitly or if there is no level defined

Reply:
Another idea how we could specify it:
If a rule has no level set, the default level is "informational". That also should work and we have no need to define either "generate" or "silent" at all.